https://github.com/RustSec/cargo-audit/commit/1734f5ea4b7a

error: unused `#[macro_use]` import
  --> src/main.rs:12:1
   |
12 | #[macro_use]
   | ^^^^^^^^^^^^
   |
note: lint level defined here
  --> src/main.rs:5:22
   |
5  | #![deny(unsafe_code, warnings, missing_docs, trivial_numeric_casts)]
   |                      ^^^^^^^^
   = note: #[deny(unused_imports)] implied by #[deny(warnings)]

error: aborting due to previous error

--- src/main.rs.orig	2018-07-29 21:14:21 UTC
+++ src/main.rs
@@ -9,6 +9,7 @@
 mod shell;
 
 extern crate gumdrop;
+#[allow(unused_imports)]
 #[macro_use]
 extern crate gumdrop_derive;
 extern crate isatty;
